packages/arb-token-bridge-ui/src/pages/api/cctp/[type].ts

```diff
@@ -237,27 +294,30 @@ export default async function handler(
       ])
     )
 
-    const { pending, completed } = messageSents.reduce(
-      (acc, messageSent) => {
-        // If the MessageSent has a corresponding MessageReceived
-        const messageReceived = messagesReceivedMap.get(messageSent.id)
-        if (messageReceived) {
-          acc.completed.push({
-            messageReceived,
-            messageSent
-          })
-        } else {
-          acc.pending.push({
-            messageSent
-          })
-        }
-        return acc
-      },
-      { completed: [], pending: [] } as {
-        completed: CompletedCCTPTransfer[]
-        pending: PendingCCTPTransfer[]
-      }
-    )
+    const { pending, completed } =
+      messagesSentToWalletAddressResult.data.messageSents
+        .concat(messagesSentFromWalletAddressResult.data.messageSents)
```

Comment:
feels like the concat can be done on a line above with a var to make it cleaner?